### ISSUE TITLE
Update org.freedesktop.Platform

### ISSUE THREAD

**Issue by paidhi** [flathub/org.davmail.DavMail#16]
Every time I check for Flatpak updates a warning is displayed (Fedora 36, OpenSUSE Tumbleweed):
```text
$ flatpak update                                                                                                                                                                                             ~
Looking for updates…
Info: org.freedesktop.Platform//20.08 is end-of-life, with reason:
   org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Applications using this runtime:
   org.davmail.DavMail
```

**Comment by PresGas** [comment]
I can confirm this with a fresh install on a new machine. The initial installation doesn't mention this fact that I recall. It is only happening with flatpak updates.

**Comment by PresGas** [comment]
Correction and apologies. The install DID also warn about the version of org.freedesktop.Platform

**Comment by wberrier** [comment]
Fixed in #19
